bench/mjolnir/buildtiles.cc

```diff
@@ -0,0 +1,41 @@
+#include <benchmark/benchmark.h>
+
+#include "baldr/rapidjson_utils.h"
+#include "mjolnir/util.h"
+
+using namespace valhalla::mjolnir;
+
+int main(int argc, char** argv) {
+  valhalla::midgard::logging::Configure({{"type", ""}});
+
+  const auto build_tiles = [](benchmark::State& state, const std::string& pbf_path,
```

Comment:
`benchmark::RegisterBenchmark` accepts Lambda type as an argument, so it's not possible to make it work with  function